rxconf/config_types.py

```diff
@@ -128,6 +136,13 @@ async def load_from_path_async(
             path=path if isinstance(path, PurePath) else PurePath(path)
         )
 
+    def _get_hash(self) -> int:
+        return self._hash
```

Comment:
In my opinion, this method is redundant.

Use self._hash directly where need and remove this method.

Reply:
This concerns all config types.